lib/teiserver/coordinator/consul_commands.ex

```diff
@@ -1129,6 +1134,17 @@ defmodule Teiserver.Coordinator.ConsulCommands do
     end
   end
 
+  defp get_default_min_players(str) do
+    case String.trim(str) do
+      "" ->
+      1
+
+      _ ->
+        String.to_integer(str |> String.trim)
+        |> max(1)
+    end
```

Comment:
Could you explain this part? As a newbie to Elixir this looks pretty magical 😅

Reply:
We get the "remaining" string from the handle command function and pass it to this.
This function is used to check if the string is empty (no min number given) by comparing the trimmed string to an empty one.
If it is empty we return 1 as a default min player number value, otherwise we convert it to integer and return the bigger value between it and 1 (in case people enter non positive numbers)

Reply:
Good explanation Lexon :)

I do have a few things to raise on the way it's written (the logic is sound, it's purely Elixir idioms).

```elixir
String.to_integer(str |> String.trim)
|> max(1)
```

I'd replace it with
```
str
|> String.trim
|> String.to_integer
|> max(1)
```

Reply:
Done